getl/blocks/transform/transform.py

```diff
@@ -75,6 +75,37 @@ def substring(
     return dataframe.withColumn(new_col, F.substring(col, pos, length))
 
 
+def split(
+    dataframe: DataFrame, col: str, new_col: str, split_on: str, item: int
+) -> DataFrame:
+    """Return DF with a column that is the result of splitting a column in a given character and getting one result back.
+
+    :param str col: name of the column
+    :param str new_col: type of the new column
+    :param int split_on: split the string on this char
+    :param int item: which item in the split array to return to the new column
+
+    Examples:
+
+    ```
+    SectionName:
+        Type: transform::generic
+        Input: InputBlock
+        Properties:
+        Functions:
+            - split:
+                col: name
+                new_col: firstname
+                split_on: ' '
+                item: 0
+    ```
+    """
+    if not _column_present(dataframe, col):
+        raise AttributeError("Column '{}' not found in df".format(col))
+
+    return dataframe.withColumn(new_col, F.split(F.col(col), split_on).getItem(item))
```

Comment:
Why not have split return a new collumn with an array? And a new function with getItem instead?

Reply:
Because this fit my usecase. But I guess splitting split and getItem is fine.